protocols/yearnfi/index.json

```diff
@@ -3,5 +3,9 @@
 	"description": "",
 	"path": "ybaby.eth",
 	"isEnabled": true,
+	"discourseForum" : {
+		"url":"https://gov.yearn.finance/",
+		"categoryID":""
```

Comment:
curiosity qn: if they have no ID, should this still be filled in? If it should be filled in anyway, how does the dashboard interpret the empty string?

Reply:
Had missed it, fixed it.